lti_emailer/settings/secure.py.j2

```diff
@@ -0,0 +1,36 @@
+SECURE_SETTINGS = {
+    'canvas_token': '{{ secure_settings.canvas_token }}',
+    'lti_oauth_credentials': {
+        '{{ secure_settings.lti_key }}': '{{ secure_settings.lti_secret }}',
+    },
+
+    'icommons_api_user': '{{ secure_settings.icommons_api_user }}',
+    'icommons_api_pass': '{{ secure_settings.icommons_api_pass }}',
+    'icommons_api_host': '{{ secure_settings.icommons_api_host }}',
+
+    'django_secret_key': '{{ secure_settings.django_secret_key }}',
+
+    'admins': (
+        ('{{ secure_settings.admin_user_name }}', '{{ secure_settings.admin_user_email }}'),
+    ),
+
+    'db_default_backend': '{{ secure_settings.db_default_backend }}',
+    'db_default_name': '{{ secure_settings.db_default_name }}',
+    'db_default_user': '{{ secure_settings.db_default_user }}',
+    'db_default_password': '{{ secure_settings.db_default_password }}',
+    'db_default_host': '{{ secure_settings.db_default_host }}',
+    'db_default_port': {{ secure_settings.db_default_port }},
+    'db_termtool_name': '{{ secure_settings.db_termtool_name }}',
+    'db_termtool_user': '{{ secure_settings.db_termtool_user }}',
+    'db_termtool_password': '{{ secure_settings.db_termtool_password }}',
+    'db_termtool_host': '{{ secure_settings.db_termtool_host }}',
+    'db_termtool_port': '{{ secure_settings.db_termtool_port }}',  # Oracle driver requires this to be a string
+
+    'cache_host': '{{ secure_settings.cache_host }}',
+    'cache_port': {{ secure_settings.cache_port }},
```

Comment:
That's fine.  I actually created a Trello card to get us away from the redis_sessions plugin and instead set the session backend to `django.contrib.sessions.backends.cache`.  I believe doing so would have the same effect of using redis_sessions and pointing to the same backend (in a different way).  The cache location uses the combined host and port, but we could easily construct that from the split out values.